### PR TITLE
Add Country detail view component in country view

### PR DESCRIPTION
**Pull request by shreeyash07**
- Addresses #5 

## Changes

- Add Image, avatar for country details
- Add details of country as shown in figma in country view

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

### CODE REVIEW

**Review by AdityaKhatri — APPROVED** [review]

File: app/views/Dashboard/Country/styles.css [hunk 1]
```diff
@@ -5,13 +5,12 @@
 
     .country-main {
         display: flex;
-        gap: var(--dui-spacing-extra-large);
+        gap: var(--dui-spacing-small);
 
         .country-detail-wrapper {
             display: flex;
-            flex-direction: column;
+            flex-flow: column;
```
Comment (subinasr) [inline app/views/Dashboard/Country/styles.css]: Why use flex-flow instead of flex-direction?

File: app/views/Dashboard/Country/styles.css [hunk 2]
```diff
@@ -30,20 +29,61 @@
...
         }
 
         .country-info {
-            flex-basis: 25%;
-            flex-shrink: 0;
             border: var(--width-separator-thin) solid var(--color-separator-light);
             border-radius: var(--dui-border-radius-card);
             background-color: var(--dui-color-foreground);
-        }
+            min-width: 20rem;
```
Comment (subinasr) [inline app/views/Dashboard/Country/styles.css]: Let's use variable rather than absolute values for widths.

**Review by subinasr — CHANGES_REQUESTED** [review]

File: app/views/Dashboard/Country/styles.css (hunk 1 above)
Comment (shreeyash07) [reply app/views/Dashboard/Country/styles.css]: Changes done

File: app/views/Dashboard/Country/styles.css (hunk 2 above)
Comment (shreeyash07) [reply app/views/Dashboard/Country/styles.css]: changes done

**Review by subinasr — APPROVED** [review]

**Pull request merged**